cogs/spawning.py

```diff
@@ -461,7 +461,22 @@ async def catch(self, ctx, *, guess: str):
 
         self.bot.dispatch("catch", ctx, species)
         await ctx.send(message)
+        
+    @checks.has_started()
+    @commands.command(aliases=("tm",))
```

Comment:
Can you remove this alias? It may be used for something else in the future

Reply:
yes